crates/hyperswitch_connectors/src/connectors/nexixpay/transformers.rs

```diff
@@ -142,87 +552,278 @@ impl<F, T> TryFrom<ResponseRouterData<F, NexixpayPaymentsResponse, T, PaymentsRe
+            authentication_value: three_d_s_complete_auth_request_data
+                .three_d_s_auth_result
+                .authentication_value,
+        };
+        let card: Result<NexixpayCard, error_stack::Report<errors::ConnectorError>> =
+            match payment_method_data {
+                PaymentMethodData::Card(req_card) => Ok(NexixpayCard {
+                    pan: req_card.clone().card_number,
+                    expiry_date: req_card.clone().get_expiry_date_as_mmyy()?,
+                    cvv: None,
+                }),
+                _ => Err(
+                    errors::ConnectorError::NotImplemented("Payment methods".to_string()).into(),
+                ),
+            };
+        Ok(Self {
+            order: order_data,
+            card: card?.clone(),
+            operation_id,
+            capture_type,
+            three_d_s_auth_data,
+        })
+    }
+}
+
+impl<F>
+    TryFrom<
+        ResponseRouterData<F, NexixpayTransactionResponse, PaymentsSyncData, PaymentsResponseData>,
+    > for RouterData<F, PaymentsSyncData, PaymentsResponseData>
+{
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        item: ResponseRouterData<
+            F,
+            NexixpayTransactionResponse,
+            PaymentsSyncData,
+            PaymentsResponseData,
+        >,
+    ) -> Result<Self, Self::Error> {
+        Ok(Self {
+            status: AttemptStatus::from(item.response.operation_result),
+            response: Ok(PaymentsResponseData::TransactionResponse {
+                resource_id: ResponseId::ConnectorTransactionId(item.response.operation_id.clone()),
+                redirection_data: None,
+                mandate_reference: None,
+                connector_metadata: None,
+                network_txn_id: None,
+                connector_response_reference_id: Some(item.response.operation_id.clone()),
+                incremental_authorization_allowed: None,
+                charge_id: None,
+            }),
+            ..item.data
+        })
+    }
+}
+
+impl TryFrom<&NexixpayRouterData<&PaymentsCaptureRouterData>> for NexixpayPaymentsCaptureRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        item: &NexixpayRouterData<&PaymentsCaptureRouterData>,
+    ) -> Result<Self, Self::Error> {
+        Ok(Self {
+            amount: item.amount.clone(),
+            currency: item.router_data.request.currency,
+        })
+    }
+}
+
+impl<F, T> TryFrom<ResponseRouterData<F, NexixpayOperationResponse, T, PaymentsResponseData>>
+    for RouterData<F, T, PaymentsResponseData>
+{
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        item: ResponseRouterData<F, NexixpayOperationResponse, T, PaymentsResponseData>,
+    ) -> Result<Self, Self::Error> {
+        Ok(Self {
+            status: item.data.status,
```

Comment:
why are we not updating status here ?

Reply:
In capture and cancel calls we don't get status in response